contracts/neutron_interchain_queries/src/msg.rs

```diff
@@ -15,6 +15,25 @@ pub enum ExecuteMsg {
         addr: String,
         denom: String,
     },
+    RegisterBankTotalQuery {
```

Comment:
`RegisterBankTotalSupplyQuery` here and everywhere else (methods, etc)?

I know it's just `total` in Cosmos SDK but I think with `supply` is a bit more obvious?